### PR TITLE
[Tram] New tram, new consequences. Tram collisions actually hurt.

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
This change updates the damaged caused from getting hit by the tram to a random number between 120 and 240, roughly a 10% chance of death.

## How This Contributes To The Skyrat Roleplay Experience
As an MRP/HRP server, characters are expected to value their bones and lives. Risking the traffic signals or riding the rails should have real consequences. In testing we've seen emergent roleplay of people doing things such as enforcement, barriers, painting if a high number of people are foolish getting hit by the tram.

## Changelog
:cl: LT3
balance: Tramstation's tram now actually packs a punch when it hits you. It will hurt. Think twice before crossing on red!
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/tram/tram_lift_master.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/datum/lift_master/tram
```
Comment (Zonespace27) [inline modular_skyrat/modules/tram/tram_lift_master.dm]: don't make a module for this, just put it in a `master_files` override

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Review by Zonespace27 — APPROVED** [review]
I like it

**Comment by Paxilmaniac** [comment]
I thought this was a nerf until I read it and realized its so, so much better

**Comment by PCPort** [comment]
Time to bucklecuff miscreants to the rails and cackle whilst twirling my moustache

**Comment by GoldenAlpharex** [comment]
I think it's a neat idea. Looking forward to see how people shape up for that change.

**Comment by Iamgoofball** [comment]
This is a terrible idea and is just going to result in you guys putting rules on the tram and further ruining an already working just fine design.

Like, alright sure, it's not a bad idea in theory but I foresee the end result being some admin banning a guy for trying to take the tram somewhere and some idiot being on the tracks.

There are times when it's easier for the health of the game as a whole to not make something realistic, and I think this is a good case of that.

**Comment by Iamgoofball** [comment]
after chatting with folks, hesitant 👍 but definitely keep an eye on it

File: modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/datum/lift_master/tram
+	collision_lethality = 4
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm:12]: Could be nice to just add a comment here saying what range of damage that equates to, just so it's easier to understand why this was done this way in the future.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm (hunk 2 above)
Comment (lessthnthree) [reply modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm:12]: Sounds good.

File: modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Damage calculation for getting hit by the tram. 120 to 240 damage to the player. A chance of death, but it's going to hurt.
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/industrial_lift/tram_lift_master.dm]:
```suggestion
 * Damage calculation for getting hit by the tram. 120 to 240 damage to the player at a lethality of 4. A chance of death, but it's going to hurt.
```

**Review by GoldenAlpharex — APPROVED** [review]
I've been happy with the results so far.

**Pull request merged**